[numerical debugger] bumped up the starting handler id

Gasoonjia opened the pull request:
Differential Revision: D65445250

dulinriley commented on torch/ao/quantization/pt2e/_numeric_debugger.py:
```diff
@@ -19,13 +19,13 @@ def generate_numeric_debug_handle(graph_module: GraphModule) -> None:
     The graph nodes of input model is modified inplace.
     """
-    unique_id = -1
+    unique_id = 0
```
Is the goal here to have the first ID start at 1 instead of 0? Why? There's no summary on this PR

Gasoonjia committed:
[numerical debugger] bumped up the starting handler id (#139666)

Summary:

This diff bumped up the starting point of debug handle created in torch.ao from 0 to 1 for fitting the pattern we use in ExecuTorch.

Test Plan: ci

Reviewed By: tarun292, dulinriley

Differential Revision: D65445250
